dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/SuggestionRestController.java

```diff
@@ -262,15 +262,18 @@ private void processSolrSearchResults(DiscoverResult searchResult, String autoco
                                           List<VocabularyEntryRest> results) {
         searchResult.getFacetResult(autocompleteCustom).forEach(facetResult -> {
             String displayedValue = facetResult.getDisplayedValue();
-            if (displayedValue.contains(searchValue)) {
-                // Create a new VocabularyEntryRest object
-                VocabularyEntryRest vocabularyEntryRest = new VocabularyEntryRest();
-                vocabularyEntryRest.setDisplay(displayedValue);
-                vocabularyEntryRest.setValue(displayedValue);
-
-                // Add the filtered value to the results
-                results.add(vocabularyEntryRest);
+            if (!displayedValue.contains(searchValue) &&
```

Comment:
This last check `!displayedValue.toLowerCase().contains(searchValue.toLowerCase())` is sufficient the previous two is redundand.